src/modules/facility-fixed-fee/dto/get-facility-fixed-fee-response.dto.ts

```diff
@@ -0,0 +1,100 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers';
+
+export type GetFacilityFixedFeeResponse = GetFacilityFixedFeeResponseItem[];
+export class GetFacilityFixedFeeResponseItem {
+  @ApiProperty({
+    description: 'The identifier of the facility.',
+    example: EXAMPLES.FACILITY_ID,
+    minLength: 10,
+    maxLength: 10,
```

Comment:
Can we add `pattern` decorator.

Reply:
updated to use `ValidatedFacilityIdentifierApiProperty` that has this pattern